docker-compose.yml

```diff
@@ -106,5 +104,5 @@ services:
      OO_SERVICE_EMAIL: "openoversightchi@lucyparsonslabs.com"
      TIMEZONE: "America/Chicago"
    volumes:
-     - ./OpenOversight/:/usr/src/app/OpenOversight/:z
+     - .:/usr/src/app:Z
```

Comment:
It might be easier to mount the database/ directory

Reply:
I added the whole directory instead of creating a volume statement for the test directory, database directory, and Makefile The thought process is that it's less fragile and we're not really worried about space on this docker container, especially since it is only for the test environment.